### ISSUE TITLE
App performance with large files

### ISSUE THREAD

**Issue by Arbiter313** [notaapp/issues#425]
I'm currently working on a markdown file around 30,000 words long and I notice the app starting to show some lag when it comes to typing. The lag seems to be limited to that particular tab, as there is no lag on a different smaller file I have open in another tab at the same time. 

Have you guys noticed this and do you guys plan to make the app more performant with large files in future updates?
Also do you have a rough ballpark in terms of how many words before the app starts to lag?

**Comment by astoilkov** [comment]
Yeah, we know about that, and yes we've been planning to improve on this. The word counts mainly depends on your computer spec so we can't tell how much words it needs before it starts lagging.
